### PR TITLE
Enhancement to allow jobs to be automatically segregated for users with too many jobs

### PR DESCRIPTION
**Pull request by rw2**

### CODE REVIEW

File: aweadmin.py [hunk 1]
```diff
@@ -211,6 +258,8 @@ def main():
       showjobs=True
     elif arg == '-n':
```
Comment (kkellerlbl) [inline aweadmin.py:241]: We should really convert these arguments to use the Python argparse module.  But that's outside the scope of this task.

File: aweadmin.py [hunk 2]
```diff
@@ -53,6 +55,50 @@ def get_jobs(baseurl,token,status):
...
+      i=d['info']
+      if not userCount.has_key(i['user']):
+        userCount[i['user']] = 0
+      userCount[i['user']] = userCount[i['user']] + 1
+
+  for user in userCount:
+    if userCount[user] > cap_limit:
+      move_to_penalty_box(baseurl, token, user, userCount[user], get_jobs(baseurl, token, "queued"), dryrun)
+
+def list_jobs(baseurl,token):
```
Comment (kkellerlbl) [inline aweadmin.py]: Are there now two different list_jobs methods?  That seems wrong.  Can you consolidate into one function?

File: aweadmin.py [hunk 3]
```diff
@@ -53,6 +55,50 @@ def get_jobs(baseurl,token,status):
   dumpdata(data)
   return data['data']
 
+def get_queues(baseurl,token,status):
```
Comment (kkellerlbl) [inline aweadmin.py]: Does this function do anything important?  I'm not familiar with the queue endpoint of AWE, and it's not used anywhere else.

File: aweadmin.py (hunk 1 above)
Comment (rw2) [reply aweadmin.py:241]: Totally agree. It kind of bugged me to not do it while I was in there, but I didn't have time to verify that there wasn't some historical reason to leave it as it was.

**Review by kkellerlbl — CHANGES_REQUESTED** [review]
I think if you can resolve the two list_jobs into one then we're ok.

File: aweadmin.py (hunk 3 above)
Comment (rw2) [reply aweadmin.py]: That was an artifact of a method I intended to write before I understood that client groups should be my focus. Good catch.

File: aweadmin.py (hunk 2 above)
Comment (rw2) [reply aweadmin.py]: No explanation for why this is here. Must have been a copy and paste error...

File: aweadmin.py (hunk 1 above)
Comment (kkellerlbl) [reply aweadmin.py:241]: No reason to leave it except laziness.  ;-)

**Comment by kkellerlbl** [comment]
Looks okay now, if you're ready I'll merge.

**Pull request merged**